en/micro-integrator/docs/references/connectors/nats-connector/nats-connector-config.md

```diff
@@ -0,0 +1,114 @@
+# NATS Connector Configuration
+[NATS](https://nats.io/) is a distributed messaging platform based on the publish-subscribe messaging model
+where publishers publish messages on a paticular subject and consumers listening on that 
+subject can consume these messages. For more information, see the [NATS documentation.](https://docs.nats.io/)
+
+The NATS connector allows you to access the NATS API through WSO2 EI and acts as a message 
+publisher that facilitates message publishing. 
+The NATS connector sends messages to NATS brokers.
+
+# Setting up the environment
+__Download and install the connector__
+
+1. Download the connector from the [WSO2 Store](https://store.wso2.com/store/assets/esbconnector/details/3fcaf309-1a69-4edf-870a-882bb76fdaa1) 
+by clicking the Download Connector button.
+
+2. You can then follow this [documentation](https://docs.wso2.com/display/EI650/Working+with+Connectors+via+the+Management+Console) to add the connector to your WSO2 EI instance 
```

Comment:
We cannot use the management console for this doc. We need to point to a doc that shows how this is done via tooling.